src/tests/Interop/NativeLibrary/NativeLibraryToLoad/GlobalLoadHelper.cpp

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#include <stdio.h>
+#include <platformdefines.h>
+#ifndef TARGET_WINDOWS
+#include <dlfcn.h>
```

Comment:
Perhaps add this to `platformdefines.h`?

Reply:
I don't think we should add this to `platformdefines.h` as we generally use that more as a PAL layer and we wouldn't be providing cross-plat wrappers around these methods.